src/test/java/com/rultor/agents/req/DecryptTest.java

```diff
@@ -133,4 +134,41 @@ public void decryptsAssets() throws Exception {
         );
     }
 
+    /**
+     * This test reproduces issue #635 and validates that Decrypt uses HTTP
+     * proxy server settings when running gpg, if they are provided.
+     * @throws java.io.IOException Thrown in case of XML parsing error
+     */
+    @Test
+    public void testHttpProxyHandling() throws IOException {
+        final Decrypt decrypt = new Decrypt(
+            new Profile.Fixed(
+                this.createTestProfileXML(),
+                TEST_PROFILE_NAME
+            ),
+            "http://someserver.com",
+            8080
+        );
+        final Iterable<String> commands = decrypt.commands();
```

Comment:
it's a one time variable, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables